### PR TITLE
Fix description of MGF

### PR DESCRIPTION
**Pull request by Julienraptor01**

### CODE REVIEW

File: Mod Menu Helper/assets/fo/lang/en_us.json [hunk 1]
```diff
@@ -42,7 +42,7 @@
   "modmenu.summaryTranslation.mixin-conflict-helper": "Alerts when two mods conflict\n§8✎ (no options)",
   "modmenu.summaryTranslation.mixintrace": "Helps developers find the cause of crashes\n§8✎ (no options)",
   "modmenu.summaryTranslation.modmenu": "This mod menu\n§9✎§8 Button placement, filtering, sorting, ...",
-  "modmenu.summaryTranslation.modelfix": "§c*§r Fix transparent lines in held items\n§8✎ (no options)",
+  "modmenu.summaryTranslation.modelfix": "§c*§r Fix transparent lines in all models\n§8✎ (no options)",
```
Comment (Madis0) [inline Mod Menu Helper/assets/fo/lang/en_us.json:45]:
```suggestion
  "modmenu.summaryTranslation.modelfix": "§c*§r Fix transparent lines in all models\n§8✎ (only advanced options)",
```

Comment (Julienraptor01) [reply Mod Menu Helper/assets/fo/lang/en_us.json:45]: I didn't think of that but it's a good idea

File: Mod Menu Helper/assets/fo/lang/en_us.json [hunk 2]
```diff
@@ -42,7 +42,7 @@
   "modmenu.summaryTranslation.mixin-conflict-helper": "Alerts when two mods conflict\n§8✎ (no options)",
   "modmenu.summaryTranslation.mixintrace": "Helps developers find the cause of crashes\n§8✎ (no options)",
   "modmenu.summaryTranslation.modmenu": "This mod menu\n§9✎§8 Button placement, filtering, sorting, ...",
-  "modmenu.summaryTranslation.modelfix": "§c*§r Fix transparent lines in held items\n§8✎ (no options)",
+  "modmenu.summaryTranslation.modelfix": "§c*§r Fix transparent lines in items and block models\n§8✎ (only advanced options)",
```
Comment (Madis0) [inline Mod Menu Helper/assets/fo/lang/en_us.json:45]: "Item and block models"? Or "items and blocks"?

Comment (Julienraptor01) [reply Mod Menu Helper/assets/fo/lang/en_us.json:45]: well item && blocks is actually enough

**Pull request merged**